spec/rubocop/cop/lint/nested_percent_literal_spec.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+describe RuboCop::Cop::Lint::NestedPercentLiteral do
+  subject(:cop) { described_class.new }
+
+  let(:message) { 'Do not nest percent literals.' }
+
+  def expect_offense(source)
```

Comment:
Why are you defining this instead of using the existing helper?

Reply:
Following the pattern [here](https://github.com/bbatsov/rubocop/blob/master/spec/rubocop/cop/lint/percent_string_array_spec.rb#L11-L16), it seems useful to define a method so as to not repeat myself throughout this file. Most likely, this should be a more-widely scoped method, though that refactoring seems appropriate for another PR.